src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    <p className="Person__partner">
+      {person.isMarried ? (
+        <>
+          {person.partnerName} is my{' '}
+          {person.sex === 'f' ? <>husband</> : <>wife</>}
```

Comment:
Nested ternary looks complicated, let's create a separate variable and use it here

`const partner = person.sex === 'f' ? 'husband' : 'wife'`